fastlane/lib/fastlane/plugins/templates/plugin.gemspec.erb

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require '<%= gem_name %>/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = '<%= gem_name %>'
+  spec.version       = Fastlane::<%= plugin_name.capitalize %>::VERSION
+  spec.author        = '<%= author %>'
+  #spec.email         = ["fastlane-xcversion@gmail.com"]
```

Comment:
Maybe we should just add our dev tools email here? Some developer might just comment it out, and then they have a valid email address where they can reach us 👍